conf/defaults.ini

```diff
@@ -371,6 +371,8 @@ client_secret = some_secret
 scopes = user:email
 email_attribute_name = email:primary
 email_attribute_path =
+role_attribute_name = role
```

Comment:
Remove and possibly move the default to a JMESPath-compatible value in `role_attribute_path` if it per default should extract role - not sure how common that attribute is in oauth and it need to correlate with Grafana roles, right. So maybe it shouldn't have a default value?